packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -140,6 +143,7 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 		otherSupportInteractions,
 		odieConversations,
 		supportInteractions,
+		lastMessageReceivedAt,
```

Comment:
We don't use that variable in the `useEffect()` clause, should we still need to define it as a dependency?

Reply:
We want to reorder the array for new data, but is true that the rest of the deps will change